libnopegl/src/backends/gl/glcontext.c

```diff
@@ -349,6 +349,8 @@ static int glcontext_probe_settings(struct glcontext *glcontext)
     GET(GL_MAX_ARRAY_TEXTURE_LAYERS, &limits->max_texture_array_layers);
     GET(GL_MAX_SAMPLES, &limits->max_samples);
     GET(GL_MAX_COLOR_ATTACHMENTS, &limits->max_color_attachments);
+    /* nope.gl only support up to 8 color attachments */
+    limits->max_color_attachments = NGLI_MIN(limits->max_color_attachments, 8);
```

Comment:
Why not use `NGLI_MAX_COLOR_ATTACHMENTS`? You wouldn't need the comment (maybe have one on the definition of the macro saying that's it's an arbitrary limit of the project)

Reply:
Sorry that was a typo. Fixed.